.travis.yml

```diff
@@ -5,8 +5,8 @@ rvm:
   - 2.3.4
   - 2.4.1
 before_install:
-  - gem update --system
-  - gem update bundler
+  - gem update --system 2.7.8
+  - gem install bundler -v '~> 1.17.3'
```

Comment:
Why are these changes necessary?

Reply:
I noticed the ruby versions in `travisCI` config are failing in the latest `rubygems` when running ``gem update --system`` and the bundler it also fails in the latest `2.0` update when ``gem update bundler`` in the previous configuration. So I need to adjust the configuration to pass the tests, you can see the test history below that fails in relation to this:
- https://travis-ci.org/krautcomputing/active_record-acts_as/builds/474851341?utm_source=github_status&utm_medium=notification
- https://travis-ci.org/krautcomputing/active_record-acts_as/builds/474856183?utm_source=github_status&utm_medium=notification

Reply:
@manuelmeurer, sure 😄

Reply:
@manuelmeurer, I've made the changes, see commit https://github.com/krautcomputing/active_record-acts_as/pull/29/commits/645364b15e388442f764b5e35a54d9e1e61b625f. Thanks.

Reply:
@manuelmeurer, it looks like ruby `2.3` also fails with rails `4.2` (https://travis-ci.org/krautcomputing/active_record-acts_as/jobs/475374092). In my opinion, we just need to support the latest stable versions of ruby (`2.4.5`, `2.5.3` and `2.6.0`) and since ruby `2.3.8` will be EOL soon acording to ruby site (https://www.ruby-lang.org/en/downloads/). What do you think?

Reply:
@manuelmeurer, If we still to support rails `4.2`, we need to limit `bundler` within `1.3` and not to go over `2.0` as rails 4 bundler dependency.